armi/reactor/reactors.py

```diff
@@ -2251,19 +2251,19 @@ def processLoading(self, cs):
 
         self.getNuclideCategories()
 
-        # some blocks will not move in the core like grid plates... Find them and fix them in place
-        stationaryBlocks = []
-        # look for blocks that should not be shuffled in an assembly.  It is assumed that the
-        # reference assembly has all the fixed block information and it is the same for all assemblies
-        for i, b in enumerate(refAssem):
-            if b.hasFlags(Flags.GRID_PLATE):
-                stationaryBlocks.append(i)
-                # TODO: remove hard-coded assumption of grid plates (T3019)
-                runLog.extra(
-                    "Detected a grid plate {}.  Adding to stationary blocks".format(b)
-                )
+        # Generate list of flags that are to be stationary during assembly shuffling
+
+        if len(cs["stationaryBlocks"]) >= 1:
+            self.ValueError(
+                "stationaryBlocks setting is deprecated. Use stationaryBlocksFlags."
+            )
```

Comment:
Maybe just remove logic that uses the `stationaryBlocks` setting since it will fail anyways.

Reply:
Removed the logic here.